content/help/faq/distributed-tracing/initial-zipkin-header.md

```diff
@@ -0,0 +1,6 @@
+---
+title: What generates the initial Zipkin (B3) HTTP headers?
+weight: 15
+---
+
+The Istio sidecar proxy (Envoy) generates the initial [headers](https://www.envoyproxy.io/docs/envoy/latest/configuration/http_conn_man/headers#x-request-id).
```

Comment:
The initial sidecar generates the headers if they are not provided by the request - i.e the interactions within the mesh maybe a continuation of a trace that is already being recorded.